DOWNLOAD_MAPS.md

```diff
@@ -1,3 +1,22 @@
+### Downloading OpenStreetMap data
+
+Use [Overpass Turbo](http://overpass-turbo.eu/), an interface for the OpenStreetMap database.
+You can run queries using [Overpass QL](https://wiki.openstreetmap.org/wiki/Overpass_API/Overpass_QL)
+and export results in `.geojson` format.
+
+The following query was used to create `map-sources/africa.json`:
+```
+area["ISO3166-1"~"GA|NG|CM|CG|CD|GQ"]->.a;
```

Comment:
cool! much better than what I created with my zero knowledge of Overpass QL. @kamilzyla @rcwhytock given that this returns a much smaller file (12MB currently), would it make sense to include entire continent? or at least boundaries of Africa? Or perhaps we could get boundaries of all continents, to give a better sense of what the user is looking at? -- but I don't know how to do that, is this something easy for you?

Reply:
Thanks! I'm no expert on Overpass QL - it took me a while to arrive at this query. Adding all African countries could be as simple as listing all their country codes; continent outlines could take up to a few hours (would have to do more research).